lib/model/CEventRatePopulationModel.cc

```diff
@@ -634,7 +634,7 @@ bool CEventRatePopulationModel::computeProbability(std::size_t pid,
 
     CAnnotatedProbabilityBuilder resultBuilder(
         result, std::max(numberAttributeProbabilities, std::size_t(1)),
-        function_t::function(gatherer.features()), gatherer.numberActivePeople());
```

Comment:
I would expect us to at least set if this is the first time we've seen a category. But if this isn't trivially accessible at the moment this could be a follow on PR.